repos/c2sm/packages/flexpart-cosmo/package.py

```diff
@@ -20,7 +20,7 @@ class FlexpartCosmo(MakefilePackage):
     depends_on('eccodes +fortran')
     depends_on('netcdf-fortran')
 
-    depends_on('eccodes@2.25:', type=('build', 'link'))
+    depends_on('eccodes', type=('build', 'link'))
```

Comment:
```suggestion
```
type=('build', 'link') is the default, so line 20 `depends_on('eccodes +fortran')` already covers this.